src/main/java/seedu/address/MainApp.java

```diff
@@ -69,25 +69,25 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from {@code storage}'s hustle book and {@code userPrefs}. <br>
```

Comment:
I think for consistency, let's use HustleBook instead of hustle book?

Reply:
ah, i think it's fine. I notice you're just following based on the original text. In that case, it's fine! 😄

Reply:
Alright!